blocks/tags/tags.js

```diff
@@ -30,6 +35,9 @@ export default async function decorate(block) {
       a.textContent = tag.name || tag.id;
       a.classList.add('button', 'primary');
       block.appendChild(a);
+      if (block.classList.contains('tagpage') && feedTags && feedTags.trim() === tag.id) {
```

Comment:
You can move the meta tags setting logic here.

Reply:
I am not sure if there is any other better way of doing this but personally i am not in favour of modifying other blocks DOM (i.e. breadcrumb) from the tags block.

Reply:
>You can move the meta tags setting logic here.
If I do so, I need to import `setMetaTag` method into feed block, because the image url comes from feed block and it's needed to be set into og:url and canonical link. That's why I put it into a common place blocks-utils.js

>I am not sure if there is any other better way of doing this but personally i am not in favour of modifying other blocks DOM (i.e. breadcrumb) from the tags block.
Agree. It's not a normal way, I had same concern.
Another option is to refactor breadcrumb block, make a call placeholder.json query and get the tag title, then set it to breadcrumb. In this way, requires one more query which may reduce page rendering performance. The current logic is that get the tag title from Tag block, then set it to breadcrumb. It reuses the Tag block query.

Reply:
> I am not sure if there is any other better way of doing this but personally i am not in favour of modifying other blocks DOM (i.e. breadcrumb) from the tags block.

Valid point. Can we enhance the `breadcrumb` block to check the `pageType` and if its `tagsPage` , do the special handling for them instead of doing it here?